pkg/test/step_integration_test_data/assert_commands/path_script/test/script.sh

```diff
@@ -0,0 +1,9 @@
+echo -n "$0 $@"
```

Comment:
IIRC `[[` is not supported by all shells, so we should probably have an explicit shebang line:

```suggestion
#!/bin/bash
echo -n "$0 $@"
```

Reply:
done thanks for the pointer